__tests__/itinerary.test.js

```diff
@@ -0,0 +1,15 @@
+const Itinerary = require('../src/itinerary')
+const Port = require('../src/port')
+
+describe('constructor', () => {
+    it('instantiates an object', () => {
+        expect(new Itinerary()).toBeInstanceOf(Object);
+    })
+
+    it('sets a ports property', () => {
+        const manchester = new Port ('Manchester');
+        const london = new Port ('London');
```

Comment:
you could replace these Ports with test doubles, as your test doesn't require them to be actual Ports. Even just assigning them to strings would work, as all your checking is if you add an array of items to the `itinerary` it assigns that array to `itinerary.ports`.